File: python/7 kyu/284.py
# https://www.codewars.com/kata/5650ab06d11d675371000003/train/python

# Passed

def split_in_parts(s, part_length): 
    sub_strings = [s[idx:idx+part_length] for idx in range(0, len(s), part_length)]
    result = " ".join(sub_strings)
    return result

output = split_in_parts("supercalifragilisticexpialidocious", 3)
print(output)

File: nodejs/8 kyu/284.js
// https://www.codewars.com/kata/5a34af40e1ce0eb1f5000036/train/javascript

// Passed

function toCsvText(array) {
  let rows = array.map((arr) => arr.join(","));
  let result = rows.join("\n");
  return result;
}

const output = toCsvText([
  [0, 1, 2, 3, 45],
  [10, 11, 12, 13, 14],
  [20, 21, 22, 23, 24],
  [30, 31, 32, 33, 34],
]);
console.log(output);